components/commercial_filter/queries/artwork.coffee

```diff
@@ -45,9 +45,16 @@ module.exports = """
     }
     sale {
       name
+      is_live_open
     }
     sale_artwork {
       lot_number
+      counts {
```

Comment:
I think we may want to check the logic on these queries-- i.e. can we optimize to _not_ query for bidder positions or sale artworks if the artwork has no `sale_id`?